versions.yaml

```diff
@@ -43,10 +43,10 @@ externals:
   description: "Third-party projects used specifically for testing"
 
   flannel:
-    url: "https://github.com/coreos/flannel"
-    version: "v0.14.0"
+    url: "https://github.com/flannel-io/flannel"
+    version: "v0.21.0"
     # yamllint disable-line rule:line-length
-    kube-flannel_url: "https://raw.githubusercontent.com/coreos/flannel/v0.14.0/Documentation/kube-flannel.yml"
+    kube-flannel_url: "https://raw.githubusercontent.com/flannel-io/flannel/v0.21.0/Documentation/kube-flannel.yml"
```

Comment:
Hi @ryansavino. As per your suggestion, I checked for the new release and updated flannel to v0.21.0. However, looks like the release has been reverted?

Reply:
Indeed... only latest release is v0.20.2. Please drop the second commit to unbreak CI @ryansavino .

Reply:
I've been discussing the 0.21.0 release tag with the flannel admins. We need this for `ppc64le` support, and would be best if we merge it now to avoid another backport.

It looks like 0.21.0 was just reposted now.